scene/resources/dynamic_font.cpp

```diff
@@ -33,9 +33,20 @@
 #include "os/file_access.h"
 #include "os/os.h"
 
-bool DynamicFontData::CacheID::operator<(CacheID right) const {
+#include <freetype/ftstroke.h>
```

Comment:
This fails building on Travis (Ubuntu 14.04) against the system freetype2, as they put freetype headers in `/usr/include/freetype2/ftstroke.h` instead of the standard `/usr/include/freetype2/freetype/ftstroke.h`.

I believe that's why freetype devs expect you to use their include macros defined in `ftheader.h`: https://www.freetype.org/freetype2/docs/reference/ft2-header_file_macros.html

IIUC, to include freetype stuff you can only refer to `ft2build.h` as entry point (which includes `ftheader.h`), and then you can use `#include FT_STROKER_H`. Here `ft2build.h` is already included in `dynamic_font.h`, so you only need the latter: https://github.com/endragor/godot/blob/3eb7eeed7968bccc79a933626418d8e9602a6023/scene/resources/dynamic_font.h#L41